docs/docs-content/release-notes/known-issues.md

```diff
@@ -16,6 +16,7 @@ The following table lists all known issues that are currently active and affecti
 | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------- | ----------------- |
+| When you upgrade VerteX from version 4.3.x to 4.4.x, a few system pods may remain unhealthy and experience _CrashLoopBackOff_ errors. This issue only impacts VMware vSphere based installations and occur because the internal Mongo DNS is incorrectly configured in the configserver ConfigMap.                                                                                                                                                                                       | Refer to the [Mongo DNS Configmap Value is Incorrect](../troubleshooting/palette-upgrade.md#mongo-dns-configmap-value-is-incorrect) troubleshooting guide for detailed workaround steps. This issue may also impact Enterprise Cluster backup operations.                                                                                                                                                                                                                                                                                                                                                       | June 24, 2024  | VerteX            |
```

Comment:
```suggestion
| When you upgrade VerteX from version 4.3.x to 4.4.x, a few system pods may remain unhealthy and experience _CrashLoopBackOff_ errors. This issue only impacts VMware vSphere-based installations and occurs because the internal Mongo DNS is incorrectly configured in the configserver ConfigMap.                                                                                                                                                                                       | Refer to the [Mongo DNS Configmap Value is Incorrect](../troubleshooting/palette-upgrade.md#mongo-dns-configmap-value-is-incorrect) troubleshooting guide for detailed workaround steps. This issue may also impact Enterprise Cluster backup operations.                                                                                                                                                                                                                                                                                                                                                       | June 24, 2024  | VerteX            |
```